packages/kitchen/src/pan/Item Groups.vue

```diff
@@ -15,24 +15,25 @@
                 md4
               >
                 <v-item>
-                  <v-card
-                    slot-scope="{ active, toggle }"
-                    :color="active ? 'primary' : ''"
-                    class="d-flex align-center"
-                    dark
-                    height="200"
-                    style="user-select: none;"
-                    @click="toggle"
-                  >
-                    <v-scroll-y-transition>
-                      <div
-                        v-if="active"
-                        class="display-3 text-xs-center"
-                      >
-                        Active
-                      </div>
-                    </v-scroll-y-transition>
-                  </v-card>
+                  <template #default="{ active, toggle }">
```

Comment:
If there's nothing else it can just be `<v-item #default="{ active, toggle }">` instead of the template